.github/workflows/tests.yml

```diff
@@ -0,0 +1,61 @@
+on:
+  workflow_call:
+name: Tests
+jobs:
+  run:
+    strategy:
+      matrix:
+        go-version: [1.16.x, 1.17.x]
```

Comment:
same here, how do we monitor these versions and keep them update (1.18.x is now out).

Reply:
Addressed it 👍